tools/cockpit.spec

```diff
@@ -523,6 +523,22 @@ getent group cockpit-wsinstance >/dev/null || groupadd -r cockpit-wsinstance
 getent passwd cockpit-wsinstance >/dev/null || useradd -r -g cockpit-wsinstance -d /nonexisting -s /sbin/nologin -c "User for cockpit-ws instances" cockpit-wsinstance
 
 %post ws
+if firewall-cmd --state; then
+    firewall-cmd --zone public --add-service cockpit
+    firewall-cmd --zone home --add-service cockpit
+    firewall-cmd --zone work --add-service cockpit
+    firewall-cmd --zone internal --add-service cockpit
+    firewall-cmd --permanent --zone public --add-service cockpit
+    firewall-cmd --permanent --zone home --add-service cockpit
+    firewall-cmd --permanent --zone work --add-service cockpit
+    firewall-cmd --permanent --zone internal --add-service cockpit
```

Comment:
These `firewall-cmd --permanent ...` calls are redundant with the `firewall-offline-cmd` calls. If you call `firewall-offline-cmd` the configuration files will be updated and `firewalld` will notice and reload them. The "watcher" takes about 5 seconds.

Reply:
Oh is it fine to use firewall-offline-cmd while firewalld is active?

Reply:
It works fine. The man page says it should only be used if firewalld is _not_ running. I guess we should adhere to that.